sumo_mgmt.py

```diff
@@ -689,6 +691,14 @@ def delete_offline_collectors():
     else:
         log('[ERROR] Successfully deleted offline Collectors')
 
+def add_Ingest_Budget(id,collectorId):
```

Comment:
nit: why is Ingest_Budget capitalized here? Seems we are using lower case functions names, should be
```
def add_ingest_budget(...)
```